.ruby-gemset

```diff
@@ -0,0 +1 @@
+storycoder-sample_app
```

Comment:
I got some help on this with Mike at the Ruby meet up. I lean red is helpful because we'll be working on a common gemset? Are there other benefits I should consider?